### DISCUSSION TITLE
Welcome to traktor Discussions!

### DISCUSSION THREAD

**Discussion by apistol78** [apistol78/traktor#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Funkyman71** [comment]
Hi, thanks for being here for Us to make things work

**Comment by Funkyman71** [comment]
Shameless, NI killed Audiio 4 interface for TRAKTOR after Monterrey (I think), it is a fantastic and now useless device.  Perhaps one of you have some idea  install it  on M1 Sonoma, I did found one for Kontrol X1 Mk1, after several years sleeping in my closet.  Well, thank you and all best regards.

**Reply by apistol78** [reply]
Hi, this repo is for Traktor *Game Engine* and not Traktor Pro (the audio software). Naming is unfortunate but the our game engine has been named Traktor long before 2000 (I have a backup on a zip drive from '95 :) )

**Comment by marcelwoj1** [comment]
You have been working on this for 30 years? very impressive

**Reply by apistol78** [reply]
Ohh, thinking about it makes me feel old :) But yes, almost 30 years. But it's been refactored, modified and rewritten a lot over the years so there is not much left from the early days.

**Reply by marcelwoj1** [reply]
It's a really well made engine, even 10 years ago on Gear Up the graphics looked good, I imagine now they are even better. That game was my childhood, that's how I found this repository. If you don't mind me asking what happened with that game?

**Reply by apistol78** [reply]
Thanks, very nice to hear! We where only a couple of guys, at most we was seven but during most of the development we where only four guys, working on Gear Up. We didn't have much money to start with and what we had went directly into development of Gear Up. Unfortunately we where very naive and didn't have a real plan on how to get enough money back to make our ends meet and to maintain the company. So when each one trying to have a family and no way for us to pay salaries, more or less forced us to look elsewhere for work.
But it was one of the most fun work experiences I have had to this day and I still wish it would have gone a different way where we could have continue working on it. 
But it makes me really happy to hear from people still remembering and appreciating Gear Up!

**Reply by marcelwoj1** [reply]
I see, well I'm glad you did decide to make the game in the end, it made a lot of people happy. You created something truly special, if you guys were to update that game I'm sure players would start flooding back even after all this time. Here are the steam reviews in the last 6 months for Gear Up, It's been 10 years!

![gearupsteamreviews](https://github.com/user-attachments/assets/0c84e826-72f5-4569-8ac5-1007a7957e57)
